fluka.sh

```diff
@@ -0,0 +1,46 @@
+package: FLUKA
+version: "%(tag_basename)s"
+tag: "2011.2c"
+source: https://gitlab.cern.ch/ALICEPrivateExternals/FLUKA.git
+requires:
+  - "GCC-Toolchain:(?!osx)"
+env:
+  FLUPRO: "$FLUKA_ROOT"
+  FLUFOR: "gfortran"
+prepend_path:
+  PATH: "$FLUKA_ROOT:$FLUKA_ROOT/flutil"
+---
+#!/bin/bash -e
+
+if [[ $(uname) != Linux || $(uname -m) != x86_64 ]]; then
```

Comment:
Don't use uname, use $ARCHITECTURE or whatever variable we used to set.

Reply:
Thanks for spotting it, I have fixed it.